tools/migration/main.go

```diff
@@ -60,13 +66,18 @@ func main() { // nolint: deadcode
 	case "-h", "--help":
 		showUsageAndExit(0)
 	case "describe", "buildonly", "migrate", "install":
-		oldRepoOpt, found := findOpt("old-repo", os.Args)
+		logFile, err := getLogFile()
+		if err != nil {
+			exitErr(err.Error())
+		}
+		logger := internal.NewLogger(logFile, getVerbose())
 
+		oldRepoOpt, found := findOpt("old-repo", os.Args)
 		if found == false {
 			exitErr(fmt.Sprintf("Error: --old-repo is required\n%s\n", USAGE))
```

Comment:
@shannonwells thinking about this more it seemed like logging these exit errors isn't what we want to do.  We want the user to see these on stderr not in the logfile by default, or stdout if running verbose.  Happy to log these if you have a different perspective but leaving off for now.

Reply:
Fine with me